### PR TITLE
fix bitcoin da test suite

### PR DESCRIPTION
**Pull request by eyusufatik**
# Description

### CODE REVIEW

File: crates/bitcoin-da/src/lib.rs [hunk 1]
```diff
@@ -9,4 +9,4 @@ pub mod verifier;
 const REVEAL_OUTPUT_AMOUNT: u64 = 546;
 
 #[cfg(feature = "native")]
-const MAX_TXBODY_SIZE: usize = 400000;
+const MAX_TXBODY_SIZE: usize = 350000;
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/lib.rs:12]: add TODO for better calc. of the number

File: crates/bitcoin-da/src/service.rs [hunk 2]
```diff
@@ -813,6 +816,7 @@ fn get_relevant_blobs_from_txs(
         }
 
         if let Ok(tx) = parse_batch_proof_transaction(&tx, rollup_name) {
+            println!("Parsed tx: {:?}", tx);
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/service.rs:819]: remove print

File: crates/bitcoin-da/src/service.rs [hunk 3]
```diff
@@ -169,6 +171,7 @@ impl BitcoinService {
                             }
                             Err(e) => {
                                 error!(?e, "Failed to send transaction to DA layer");
+                                println!("Failed to send transaction to DA layer: {:?}", e);
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/service.rs:174]: remove print

File: crates/bitcoin-da/src/service.rs [hunk 4]
```diff
@@ -1073,6 +1252,7 @@ mod tests {
             "Publickey recovered incorrectly!"
         );
 
+        println!("{:?}", incorrect_pub_key);
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/service.rs:1255]: remove print

File: crates/bitcoin-da/src/verifier.rs [hunk 5]
```diff
@@ -180,49 +182,49 @@ impl DaVerifier for BitcoinVerifier {
...
-                    .starts_with(WITNESS_COMMITMENT_PREFIX)
-            });
-            match commitment_idx {
-                // If commitmet does not exist
-                None => {
-                    // Relevant txs should be empty if there is no wtiness data because data is inscribed in the witness
-                    if !blobs.is_empty() {
-                        return Err(ValidationError::InvalidBlock);
-                    }
+        // if !completeness_proof.is_empty() {
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/verifier.rs:185]: remove

Comment (eyusufatik) [reply crates/bitcoin-da/src/verifier.rs:185]: @ercecan do you know why we had this if?

File: crates/bitcoin-da/src/verifier.rs [hunk 6]
```diff
@@ -840,6 +819,8 @@ mod tests {
 
         completeness_proof.clear();
 
+        println!("{:?}", completeness_proof);
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/verifier.rs:822]: remove

File: crates/bitcoin-da/src/helpers/test_utils.rs [hunk 7]
```diff
@@ -56,104 +49,79 @@ pub(crate) fn get_mock_data() -> (
...
+    // parse_transaction(&block_txs[3], "sov-btc").unwrap();
+
+    // for (id, tx) in block_txs.iter().enumerate() {
+    //     let r = parse_transaction(tx, "sov-btc");
+    //     let err = if let Err(e) = r {
+    //         e
+    //     } else {
+    //         ParserError::ScriptError("OK".to_string())
+    //     };
+    //     dbg!(id, err);
```
Comment (eyusufatik) [inline crates/bitcoin-da/src/helpers/test_utils.rs:88]: remove

**Review by kpp — APPROVED** [review]

**Pull request merged**